client/Combatant/Combatant.ts

```diff
@@ -309,4 +309,70 @@ export class Combatant {
       .filter(c => c.IsPlayerCharacter() === this.IsPlayerCharacter())
       .sort((a, b) => a.Initiative() - b.Initiative())[0];
   }
+
+  private AutoPopulatedNotes: () => string = () => {
+    let notes = "";
```

Comment:
I think it would be a little cleaner if you started notes as a `string []`, pushed members into it as you go, and then call `.join("\n\n");` at the end. String concatenation is a little hazardous IMO.